flashinfer_trace/tests/references/test_fused_add_rmsnorm_h3584.py

```
@@ -0,0 +1,123 @@
+    """Run comprehensive tests for Fused Add+RMSNorm h3584."""
+    print("Testing Fused Add+RMSNorm h3584 (Qwen2.5 7B) Reference Implementation")
+
+    test_configs = [1, 4, 8, 16, 32]
+    atol, rtol = 8e-3, 1e-2
+
+    passed = 0
+    for batch_size in test_configs:
+        try:
+            if test_correctness(batch_size, atol, rtol):
+                passed += 1
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)